lib/media.js

```diff
@@ -274,6 +274,20 @@ function getStyleFiles(state) {
     }
   }
 
+  // check for clay compile linked? clay compile font files?
```

Comment:
would you mind changing this comment to something about checking for linked fonts?